src/DevBetterWeb.UploaderApp/UploaderManager.cs

```diff
@@ -80,6 +90,31 @@ public async Task SyncAsync(string folderToUpload)
       }
     }
 
+    private async Task<AnimatedThumbnailsResponse> CreateAnimatedThumbnails(long videoId)
+    {
+      Console.WriteLine($"Creating Animated Thumbnails");
+
+      Thread.Sleep(60 * 1000);
```

Comment:
This is pretty slow - are we trying to give Vimeo a chance to process the video? Is there a way to get a callback that it's ready (or poll it periodically) instead of waiting a fixed period of time?